### DISCUSSION TITLE
Unable to install neuralprophet in a conda environment

### DISCUSSION THREAD

**Discussion by gaurav1195** [ourownstory/neural_prophet#324] [Q&A - get help using NeuralProphet]
Hi

I am able to install neural prophet when a conda environment is not activated, but when I activate a conda env and try to pip install it there it throws error

ERROR: Could not find a version that satisfies the requirement neuralprophet
ERROR: No matching distribution found for neuralprophet

Can somebody please help?

**Comment by gaurav1195** [comment]
Could it be because the python version in the environment is 3.6.13?

**Reply by ourownstory** [reply]
It may be, though it should work with 3.6.9 - If you pull the github repo and install from there (not from pypi).
